symbolic-cabi/src/sourcemapcache.rs

```diff
@@ -156,6 +158,7 @@ fn make_token_match(token: SourceLocation, context_lines: u32) -> *mut SymbolicS
         line: token.line() + 1,
         col: token.column() + 1,
         src: SymbolicStr::new(token.file_name().unwrap_or_default()),
+        name: SymbolicStr::new(token.name().unwrap_or_default()),
```

Comment:
The `unwrap_or_default` means we will return an empty string in case there is no name. is that really what we want?

Reply:
Yes, because `decode_str` will then return `''`, which then will fallback to `None` in `sourcemapcache.py`